tests/test_bindings.py

```diff
@@ -1632,3 +1637,50 @@ def test_try_star_binding_walrus_exeption_type():
         """
     )
     assert list(get_bindings(node)) == ["a", "grp_type", "grp", "b", "c", "d"]
+
+
+@type_parameter_syntax
+def test_type_var_bindings():
+    node = _parse("type RecursiveList[T] = T | list[RecursiveList[T]]")
+    assert list(get_bindings(node)) == ["RecursiveList"]
```

Comment:
I'm surprised that this all just worked without any changes to `_bindings.py`.  Nice.

Reply:
Yep. “Type aliases can refer to themselves” (as defined in the PEP) works because _get_requirements_for_type_alias does `scope.add(node.name.id)`.

Reply:
The only new binding introduced here is for `TypeAlias`es, which have a `Name` and is resolved by the existing `_get_bindings_for_name` function. Type parameters, which are `TypeVar | ParamSpec | TypeVarTuple`, are treated for `TypeAlias | FunctionDef | AsyncFunctionDef | ClassDef` as `arguments` are treated for `FunctionDef | AsyncFunctionDef | Lambda`: they are added into the current scope for their usages to be resolved.